src/styles/blocks/catalog-page.scss

```diff
@@ -0,0 +1,7 @@
+.catalog-page {
+  max-width: 1200px;
```

Comment:
Do we actually need to limit max-width? It looks strange on wide screens (too many blank space on the right)

![1](https://user-images.githubusercontent.com/66570142/113427894-877bea80-93de-11eb-89af-ea677a61645c.png)